tests/test_visualization.py

```diff
@@ -0,0 +1,85 @@
+import torch
+import nibabel as nib
+import numpy as np
+import pytest
+from PIL import Image, ImageChops
+import matplotlib
+import os
+from liver_imaging_analysis.engine.config import config
+config.visualization['volume'] = 'tests/testdata/data/volume/resized_liver.nii'
+config.visualization['mask'] = 'tests/testdata/data/mask/resized_mask.nii'
+import liver_imaging_analysis.engine.visualization as vs
+
+
+
+# Prevents matplotlib from opening figure windows
+matplotlib.use("Agg")
+
+
+@pytest.fixture
+def volume_path():
+    return config.visualization['volume']
+
+
+@pytest.fixture
+def mask_path():
+    return config.visualization['mask']
+
+
+@pytest.fixture
+def volume_data(volume_path):
+    volume_data = nib.load(volume_path).get_fdata()
+    return  torch.tensor(volume_data)
+
+
+@pytest.fixture
+def mask_data(mask_path):
```

Comment:
Too many unnecessary fixtures, you can replace them with one or two fixtures only.